src/applications/personalization/dashboard/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -301,7 +301,7 @@ describe('ApplyForBenefits component', () => {
           fetchCalls.some(call => {
```

Comment:
Minor nit, but shouldn't the comment now say 'make sure we are NOT fetching DD4EDU info' ? Otherwise looks good

Reply:
yep sorry will fix